Traceback (most recent call last):
  File "C:\Users\TMillaKoch\Anaconda3\envs\test\lib\site-packages\jupyter_cache\executors\utils.py", line 56, in single_nb_execution
    record_timing=False,
  File "C:\Users\TMillaKoch\Anaconda3\envs\test\lib\site-packages\nbclient\client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "C:\Users\TMillaKoch\Anaconda3\envs\test\lib\site-packages\nbclient\util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "C:\Users\TMillaKoch\Anaconda3\envs\test\lib\site-packages\nbclient\util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "C:\Users\TMillaKoch\Anaconda3\envs\test\lib\asyncio\base_events.py", line 568, in run_until_complete
    return future.result()
  File "C:\Users\TMillaKoch\Anaconda3\envs\test\lib\site-packages\nbclient\client.py", line 664, in async_execute
    cell, index, execution_count=self.code_cells_executed + 1
  File "C:\Users\TMillaKoch\Anaconda3\envs\test\lib\site-packages\nbclient\client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "C:\Users\TMillaKoch\Anaconda3\envs\test\lib\site-packages\nbclient\client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
import numpy as np
from IPython.display import display
import pandas as pd

data = pd.read_csv('tables\Strategy1.csv', index_col=False)

def fix_table(val):
    return str(val)

def highlighttab7(val):
    red = '#ff0000;'
    yellow = '#ffff00;'
    lgreen = '#92d050;'
    dgreen = '#03853e;'

    if val=="Medium" or val=="Need more information": color = yellow
    elif val=="Very high" or val=="Very effective" : color = lgreen
    elif val =="High" or val=="Effective": color = dgreen
    else: color = 'white'
    return 'background-color: %s' % color

data = data.replace(np.nan, '', regex=True)

data = data.applymap(fix_table)

data.style.applymap(highlighttab7).hide(axis='index')
------------------

[1;31m---------------------------------------------------------------------------[0m
[1;31mAttributeError[0m                            Traceback (most recent call last)
[1;32m~\AppData\Local\Temp\ipykernel_764\146193926.py[0m in [0;36m<module>[1;34m[0m
[0;32m     24[0m [0mdata[0m [1;33m=[0m [0mdata[0m[1;33m.[0m[0mapplymap[0m[1;33m([0m[0mfix_table[0m[1;33m)[0m[1;33m[0m[0m
[0;32m     25[0m [1;33m[0m[0m
[1;32m---> 26[1;33m [0mdata[0m[1;33m.[0m[0mstyle[0m[1;33m.[0m[0mapplymap[0m[1;33m([0m[0mhighlighttab7[0m[1;33m)[0m[1;33m.[0m[0mhide[0m[1;33m([0m[0maxis[0m[1;33m=[0m[1;34m'index'[0m[1;33m)[0m[1;33m[0m[0m
[0m
[1;31mAttributeError[0m: 'Styler' object has no attribute 'hide'
AttributeError: 'Styler' object has no attribute 'hide'

